interface/src/avatar/MyAvatar.h

```diff
@@ -283,15 +283,17 @@ class MyAvatar : public Avatar {
      *     the value.</p>
      * @property {number} analogPlusSprintSpeed - The sprint (run) speed of your avatar for the "AnalogPlus" control scheme.
      * @property {MyAvatar.SitStandModelType} userRecenterModel - Controls avatar leaning and recentering behavior.
-     * @property {number} isInSittingState - <code>true</code> if the user wearing the HMD is determined to be sitting
-     *     (avatar leaning is disabled, recentering is enabled), <code>false</code> if the user wearing the HMD is
-     *     determined to be standing (avatar leaning is enabled, and avatar recenters if it leans too far).
-     *     If <code>userRecenterModel == 2</code> (i.e., "auto") the property value automatically updates as the user sits
-     *     or stands, unless <code>isSitStandStateLocked == true</code>. Setting the property value overrides the current
-     *     sitting / standing state, which is updated when the user next sits or stands unless
-     *     <code>isSitStandStateLocked == true</code>.
+     *     <p class="important">Deprecated: This property is deprecated and will be removed. If you need it, please contact 
+     *     the developers.</p>
```

Comment:
```suggestion
     *     <p class="important">Deprecated: This property is deprecated and will be removed.</p>
```
Don't need to say "contact the developers", here an elsewhere.